internal/provider/resource_integer.go

```diff
@@ -3,138 +3,184 @@ package provider
 import (
 	"context"
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
```

Comment:
Super nit: Typically imports outside the standard library should be in a separate import group like it was previously. Tooling like `goimports` will automatically do this.

Reference: https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Looks like I forgot to configure `goimports` in GoLand. Fixed